src/main/resources/data/computercraft/lua/rom/programs/edit.lua

```diff
@@ -47,6 +47,31 @@ else
     stringColour = colours.white
 end
 
+local runHeader = [[local current = term.current()
+local ok,err = load([===============]] .. [[==================[]]
+local runHandler = [[]===============]] .. [[==================], "@.temp", nil, _ENV)
+if ok then ok,err = pcall(ok,...) end
+term.redirect(current)
+term.setTextColor(%i)
+term.setBackgroundColor(%i)
+term.setCursorBlink(false)
+local _,y = term.getCursorPos()
+local _,h = term.getSize()
+if err then
+    printError(err)
+end
+if not err and y >= h then
+    term.scroll(1)
+end
+term.setCursorPos(1,h)
+if not err then
+    write("Execution finished. ")
```

Comment:
```suggestion
    write("Program finished. ")
```

Up to you - IMO it's a little friendlier?